src/parser/vast_parser.js

```diff
@@ -209,6 +211,20 @@ export class VASTParser extends EventEmitter {
       }
     }
 
+    const vastAdsLength = vastResponse.ads.length;
+    const lastAddedAds = vastResponse.ads[vastAdsLength - 1];
```

Comment:
what do you think about:
```Javascript
const lastAddedAdIndex = vastResponse.ads.length - 1;
const lastAddedAd = vastResponse.ads[lastAddedAdIndex];
```

Anyway no big opinion, choose the one you prefer :p